### PR TITLE
Add some open source music players

### PR DESCRIPTION
**Pull request by linsui**
Update Winyl: Add 64bit architecture,
Add Exaile 4.0.0,
Add Quod Libet 4.2.1,
Add Museeks 0.11.0,
Add Musique 1.6,
Add Audacious 3.10.1,
Add Strawberry Music Player 0.5.5

### CODE REVIEW

File: bucket/audacious.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://audacious-media-player.org/",
+    "description": "An open source audio player.",
+    "license": "BSD-2-Clause",
+    "version": "3.10.1",
+    "url": "https://distfiles.audacious-media-player.org/audacious-3.10.1-win32.zip",
+    "hash": "f8fa25bdfb9775c23a9373aa2220fbf0693aab315350ee8e38720ad068194f18",
+    "bin": [
```
Comment (Ash258) [inline bucket/audacious.json]:
```suggestion
    "bin": [
        "bin\\audacious.exe",
        "bin\\audtool.exe"
    ],
```

File: bucket/museeks.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "shortcuts": [
+        [
+            "Museeks.exe",
+            "Museeks"
+        ]
+    ],
+    "architecture": {
+        "64bit": {
+            "installer": {
+                "script": [
```
Comment (Ash258) [inline bucket/museeks.json]: Run `.\bin\formatjson` or checkver to convert into strings

File: bucket/museeks.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        },
+        "32bit": {
+            "installer": {
+                "script": [
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\""
+                ]
+            }
+        }
+    },
+    "post_install": [
```
Comment (Ash258) [inline bucket/museeks.json]:
```suggestion
    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall Museeks.exe\" -Recurse",
```

File: bucket/quodlibet.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            "Quod Libet"
+        ],
+        [
+            "data\\bin\\exfalso.exe",
+            "Ex Falso"
+        ]
+    ],
+    "persist": "config",
+    "checkver": {
+        "url": "https://github.com/quodlibet/quodlibet/releases/latest",
```
Comment (Ash258) [inline bucket/quodlibet.json]:
```suggestion
        "github": "https://github.com/quodlibet/quodlibet",
```

**Comment by Ash258** [comment]
I really need to finish PR actions as soon as possible. I am too lazy to review all checkvers, autoupdates and hashes :D

**Comment by linsui** [comment]
Yeah, it's tiring. Before pr I have check them for several times but still miss something...

**Comment by Ash258** [comment]
I do not know about official buckets. In official buckets i follow majority. Add bin all the times. 

For my personal buckets I add only if there are some good command line options.

**Comment by linsui** [comment]
I got it, thanks. :ok_hand:

File: bucket/museeks.json (hunk 3 above)
Comment (Ash258) [reply bucket/museeks.json]: Not resolved

Comment (linsui) [reply bucket/museeks.json]: Sorry, I missed it...

**Pull request merged**